northd/docs/new-feature-tutorial.md

```diff
@@ -0,0 +1,392 @@
+# Adding a new OVN feature to the DDlog version of ovn-northd
+
+This document describes the usual steps an OVN developer should go through
+when adding a new feature to `ovn-norhtd-ddlog`. In order to make things
+less abstract we will use the IP Multicast `ovn-northd-ddlog` implementation
+as an example. Even though the document is structured as a tutorial there
+might still exist feature specific aspects that are not covered here.
+
+A new feature in `ovn-northd` usually has (some of) the following components:
+1. Read feature specific configuration from the `OVN Northbound` database.
+1. If the feature needs special operations from `ovn-controller`, populate
```

Comment:
I wonder if this would be a slightly more accurate description:

"If the feature affects the `ovn-controller` configuration, populate feature-specific configuration tables/columns in `OVN Southbound`"

Reply:
Sounds better indeed. I'll update it, thanks!